apps/src/applab/designElements/EnumPropertyRow.jsx

```diff
@@ -0,0 +1,38 @@
+var rowStyle = require('./rowStyle');
+
+var EnumPropertyRow = React.createClass({
+  propTypes: {
+    initialValue: React.PropTypes.string.isRequired,
+    options: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
+    handleChange: React.PropTypes.func
```

Comment:
Your code below assumes that `handleChange` is provided - maybe just make this required for now?